### PR TITLE
Fixed argument spec for multiple modules

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

This change contains fixes for argument spec and respective datatypes.

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE 
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/spec_fix.yml

### CODE REVIEW

**Comment by cben** [comment]
Great work!
shipit

**Review by mwperina — APPROVED** [review]
+1

File: lib/ansible/modules/monitoring/pagerduty_alert.py [hunk 1]
```diff
@@ -191,7 +191,7 @@ def main():
             name=dict(required=False),
             service_id=dict(required=True),
             service_key=dict(require=False),
```
Comment (felixfontein) [inline lib/ansible/modules/monitoring/pagerduty_alert.py]: Here's another one:
```suggestion
            service_key=dict(required=False),
```

**Review by felixfontein — APPROVED** [review]
Looks good. All these changes (including the one I added) are safe, since they set the parameters to their default values.

**Review by felixfontein — APPROVED** [review]
LGTM

I've retriggered the failed test.

**Comment by Akasurde** [comment]
@cben @mwperina @felixfontein Thanks for the reviews.

**Comment by felixfontein** [comment]
@Akasurde thanks for fixing this!

**Pull request merged**